docs/installation/server-config/config-reference.rst

```diff
@@ -371,11 +372,9 @@ When deploying KSQL to production, the following settings are recommended in you
     # Kafka cluster is unavailable.
     ksql.streams.producer.max.block.ms=9223372036854775807
 
-    # Set the replication factor for internal topics, the command topic, and
-    # output topics to be 3 for better fault tolerance and durability. Note:
```

Comment:
I like having the rationale "for better fault tolerance and durability" in the comment, to help users understand why these configs are valuable. Curious to know why you've removed it?

Reply:
Good point, added the rationale for the config too.